src/release.rs

```diff
@@ -85,6 +95,23 @@ impl BuildConfig {
         Ok(())
     }
 
+    async fn setup_tar_deploy(&self, file_name: &str) -> Result<PathBuf, Box<dyn Error>> {
+        let tar_file = format!("{}{}", file_name, ".tar.bz2");
```

Comment:
nit: you could simplify this to
```suggestion
        let tar_file = format!("{file_name}.tar.bz2");
```